just_audio_platform_interface/lib/just_audio_platform_interface.dart

```diff
@@ -1327,6 +1636,113 @@ abstract class AudioEffectMessage {
   Map<dynamic, dynamic> toMap();
 }
 
+class DarwinDelayMessage extends AudioEffectMessage {
+  final double delayTime;
+  final double feedback;
+  final double lowPassCutoff;
+  final double wetDryMix;
+
+  DarwinDelayMessage({
+    required bool enabled,
+    required this.delayTime,
+    required this.feedback,
+    required this.lowPassCutoff,
+    required this.wetDryMix,
+  }) : super(enabled: enabled);
+
+  @override
+  Map<dynamic, dynamic> toMap() => <dynamic, dynamic>{
+        'type': 'DarwinDelay',
+        'enabled': enabled,
+        'delayTime': delayTime,
+        'feedback': feedback,
+        'lowPassCutoff': lowPassCutoff,
+        'wetDryMix': wetDryMix,
+      };
+}
+
+enum DarwinReverbPreset {
+  smallRoom,
+  mediumRoom,
+  largeRoom,
+  mediumHall,
+  largeHall,
+  plate,
+  mediumChamber,
+  largeChamber,
+  cathedral,
+  largeRoom2,
+  mediumHall2,
+  mediumHall3,
+  largeHall2,
+}
+
+class DarwinReverbMessage extends AudioEffectMessage {
+  final double wetDryMix;
+  final DarwinReverbPreset? preset;
+
+  DarwinReverbMessage({
+    required bool enabled,
+    required this.wetDryMix,
+    this.preset,
+  }) : super(enabled: enabled);
+
+  @override
+  Map<dynamic, dynamic> toMap() => <dynamic, dynamic>{
+        'type': 'DarwinReverb',
+        'enabled': enabled,
+        'wetDryMix': wetDryMix,
+        'preset': preset?.index
```

Comment:
Sarebbe meglio passare il nome del valore dell'enum, in caso di debug sarebbe più chiaro